src/sass/_utils.scss

```diff
@@ -0,0 +1,33 @@
+// Utils are meant to override other styles (See 7-1 or ITCSS for reference)
+// sass-lint:disable no-important
+
+// Space between (Lobotomized owl) util for controlling the space between child elements.
+// You can take the child out of the layout area by giving the child a .sg-space-ignore class.
+// To automaticaly ignore empty children use "empty" class variant e.g. empty:sg-space-y-m
+@each $breakpoint, $variant in $responsiveVariants {
```

Comment:
@coderitual I'm just wondering if we should add some docs on styleguide.brainly.com about how to use those classes? WDYT?

Reply:
@Malinicz I totally agree. For sure there will be section in our new storybook docs, but maybe it is also worth putting docs into our basic section of current style guide. Let me do this in separate PR.